Diana20920/sweater-weather
# Revisit section one

### Completed
- [x] Account for bad params in Forecast and Background Controllers
- [x] Create `invalid_params` in Application Controller

### Tests 
- Add sad path/edge case in Forecast and Background request specs
- Forecast request spec can expect Float or Integer
- Add `:vcr` to Map Service spec 

_Current Coverage:_ `100.0%`

### What's Next 
- Revisit section two

#### Issues
Closes #8
